packages/components/input/__tests__/input.test.tsx

```diff
@@ -180,6 +180,26 @@ describe("Input", () => {
 
     expect(inputs[1]).toBeVisible();
   });
+
+  it("should not show clear button when isReadOnly is true", () => {
+    const {queryByRole} = render(
+      <Input isClearable isReadOnly defaultValue='This is "isReadOnly"' label="test input" />,
+    );
+
+    const clearButton = queryByRole("button");
+
+    expect(clearButton).toBeNull();
+  });
+
+  it("should show clear button when isReadOnly is false", () => {
+    const {getByRole} = render(
+      <Input isClearable defaultValue='This is "isReadOnly"' label="test input" />,
```

Comment:
The statement `This is "isReadOnly"` seems not true in this case

Reply:
Thank you for your feedback!
I have written new test code.